ClientCore/SavedGameManager.cs

```diff
@@ -76,10 +76,7 @@ public static List<string> GetSaveGameTimestamps()
 
         public static bool AreSavedGamesAvailable()
         {
-            if (Directory.Exists(SaveGameDirectoryPath))
-                return true;
-
-            return false;
+            return Directory.Exists(SaveGameDirectoryPath);
```

Comment:
Should use expression body here.

Reply:
I'll fix it.
And you can make an .editorconfig file to constraint it.